helm/nessie/ci/inmemory-values.yaml

```diff
@@ -4,3 +4,64 @@ tracing:
   enabled: true
 service:
   sessionAffinity: ClientIP
+catalog:
+  enabled: true
+  iceberg:
+    defaultWarehouse: warehouse1
+    configDefaults:
+      default1: value1
+      default2: value2
+    configOverrides:
+      override1: value1
+      override2: value2
+    warehouses:
+    - name: warehouse1
+      location: s3://bucket1.prod-us/warehouse
+      configDefaults:
+        default1: value11
+        default2: value22
+      configOverrides:
+        override1: value11
+        override2: value22
+  storage:
+    s3:
+      region: us-west-2
+      endpoint: s3.amazonaws.com
+      transport:
+        readTimeout: PT1M
+      buckets:
+        - name: bucket1.prod-us
+          endpoint: prod-us.s3.amazonaws.com
+          region: us-east-1
+#          accessKeySecret:
```

Comment:
Need to create the secrets beforehand, I suggest that we look into it in another PR.